pilot/pkg/security/model/authentication.go

```diff
@@ -52,8 +52,8 @@ const (
 	// Binary header name must has suffix "-bin", according to https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md.
 	K8sSAJwtTokenHeaderKey = "istio_sds_credentials_header-bin"
 
-	// GatewaySdsUdsPath is the UDS path for ingress gateway to get credentials via SDS.
-	GatewaySdsUdsPath = "unix:./var/run/ingress_gateway/sds"
+	// CredentialNameSDSUdsPath is the UDS path for ingress gateway to get credentials via SDS.
+	CredentialNameSDSUdsPath = "unix:./var/run/ingress_gateway/sds"
```

Comment:
I think we should change “unix:./var/run/ingress_gateway/sds” in the future as well.

Reply:
We cannot change without some painful migration. since we are moving SDS to istiod I would rather just not bother?

Reply:
Sure.